src/psyclone/tests/nemo/nemo_psy_test.py

```diff
@@ -381,7 +386,8 @@ def test_empty_routine():
     contain any executable statements. '''
     psy, _ = get_invoke("empty_routine.f90", api=API, idx=0)
     assert len(psy.invokes.invoke_list) == 1
-    assert psy.invokes.invoke_list[0].schedule is None
+    # We should just have an empty schedule
```

Comment:
I'm happy to change to an empty schedule rather than returning None if you are. If so you can ignore my previous comment about putting the check for an empty routine back in the code.

Reply:
Great.